src/System.Text.Json/src/System/Text/Json/Serialization/JsonClassInfo.AddProperty.cs

```diff
@@ -9,6 +9,11 @@ namespace System.Text.Json.Serialization
 {
     internal partial class JsonClassInfo
     {
+        private JsonPropertyInfo AddPolicyProperty(Type propertyType, JsonSerializerOptions options)
+        {
+            return AddProperty(propertyType, propertyInfo : null, classType : propertyType, options);
+
+        }
         private JsonPropertyInfo AddProperty(Type propertyType, PropertyInfo propertyInfo, Type classType, JsonSerializerOptions options)
         {
             JsonPropertyInfo jsonInfo = CreateProperty(propertyType, propertyType, propertyInfo, classType, options);
```

Comment:
Up to you, but imo, passing `propertyType` here twice, makes it hard to reason about what this method is doing.

Reply:
I'll add some comments. The `classType` used the second time is really a dummy argument (can't be null) - I'll change it to typeof(object) and comment.